### DISCUSSION TITLE
Help about project.

### DISCUSSION THREAD

**Discussion by JavaPassioner** [vLuckyyy/HelloWorldApplications#5] [General]
Is the project continuing?
What languages can I add in PR?
Do you have any TODO?

**Comment by vLuckyyy** [comment]
Hi, the project continues in my spare time, I am doing something here, you can add any languages you want. I don't have a TODO

**Reply by JavaPassioner** [reply]
> Hi, the project continues in my spare time, I am doing something here, you can add any languages you want. I don't have a TODO

Thanks
